doc/source/docs/howto_build_docs.rst

```diff
@@ -5,7 +5,7 @@ Building the NumPy API and reference docs
 =========================================
 
 We currently use Sphinx_ for generating the API and reference
-documentation for NumPy.  You will need Sphinx 1.8.3 <= 1.8.5.
+documentation for NumPy.  You will need Sphinx >= 2.0.0.
```

Comment:
I felt bad about doing this only for latex PDF, but it seems testing (I don't know if that includes building docs, and possibly it does it only for html) goes via .circleci/config.yml which has ``sphinx==2.2.0``

Reply:
There were problems with sphinx 2.0.0, we should document what we use in the circle CI html run: 2.2.0